dirty_cat/test/test_string_distances.py

```diff
@@ -22,6 +23,19 @@ def _random_string_pairs(n_pairs=50):
         pairs.append((s1, s2))
     return pairs
 
+def _random_common_char_pairs(n_pairs=50):
+    """
+    Return string pairs with a common char at random positions. This should
+    discriminate different thresholds for matching chararacters in Jaro distance.
+    """
+    # Make strings with random length and common char at index 0
```

Comment:
You should not use the global random number generator, as this leads to tests that are not perfectly reproducible. Rather, create a local random number generator.
```suggestion
    # Make strings with random length and common char at index 0
    rng = np.random.RandomState(seed=seed)
```